src/GUI/Pages/ProjectTask/Tasks.razor

```diff
@@ -14,8 +14,36 @@
             class="nav-link" 
             data-toggle="modal"
             data-target="#сreate-task-modal-center">
-                @Resources.CreateNewTask
+                Create new work item
```

Comment:
why do you not use @Resources variables?

Reply:
Fixed